sources/packages/web/src/services/http/dto/ScholasticStanding.dto.ts

```diff
@@ -5,7 +5,7 @@ export const INVALID_OPERATION_IN_THE_CURRENT_STATUS =
   "INVALID_OPERATION_IN_THE_CURRENT_STATUS";
 
 export interface ScholasticStandingDataAPIInDTO {
```

Comment:
This DTO is still an interface?

Reply:
There is one more DTO which extends this and its an interface. 
![image](https://github.com/bcgov/SIMS/assets/54600590/c6c48b87-5b39-422d-8ac5-c928bf0edef6)

Reply:
Only DTOs that will have their type enforced were changed to a class to have the class transformation possible.

Reply:
Ah I see. I was under the impression that we will be starting to use class instead of interface for the DTOs in Vue side.